internal/provider/resource_gitlab_group_test.go

```diff
@@ -246,6 +246,33 @@ func TestAccGitlabGroup_disappears(t *testing.T) {
 	})
 }
 
+func TestAccGitlabGroup_PreventForkingOutsideGroup(t *testing.T) {
+	var group gitlab.Group
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckGitlabGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				SkipFunc: isRunningInCE,
+				Config:   testAccGitlabGroupPreventForkingOutsideGroupConfig(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGitlabGroupExists("gitlab_group.foo", &group),
+					func(s *terraform.State) error {
```

Comment:
Just noticed this now, but couldn't you use `resource.TestCheckResourceAttrPtr` instead of the custom check function?

In addition, it would be nice to test this fields update logic.

Reply:
Thank you for the pointer. I'm still new to the provider SDK, I guess I should take a look at the tools available to me 😄 .